src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -833,13 +833,16 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		}
 	}
 
-	private createCell(cellType: CellType): ICellModel {
+	private createCell(cellType: CellType, language?: string): ICellModel {
```

Comment:
Please add some notebook model unit tests for this.

Reply:
Done.